.pre-commit-config.yaml

```diff
@@ -8,8 +8,9 @@ repos:
         name: black
         language: system
         entry: black
+        minimum_pre_commit_version: 2.9.0
```

Comment:
2.9.2 would be ever so slightly better -- there was some breaking in 2.9.0-flat related to types_or

Reply:
Thanks for reviewing and suggesting this. Updated in #1895.